package.json

```diff
@@ -44,16 +44,11 @@
     "@types/meow": "^5.0.0",
     "@types/node": "^14.14.32",
     "@types/prettier": "^1.18.0",
-    "colors": "^1.3.3",
     "contentful": "^8.1.9",
-    "coveralls": "^3.0.5",
-    "cross-env": "^5.2.0",
```

Comment:
https://www.npmjs.com/package/cross-env

Very interesting and possibly useful plugin, but it's not solving a specific problem for this repo. I think this should be installed by the Windows users as a global package instead if they have problems with their node apps 😃